Sources/PostgresNIO/Utilities/PostgresJSONDecoder.swift

```diff
@@ -22,7 +22,6 @@ extension PostgresJSONDecoder {
     }
 }
 
-//@available(macOS 13.0, iOS 16.0, tvOS 16.0, watchOS 9.0, *)
```

Comment:
There is a Sendable warning for these 2 JSON-Coder conformances. I didn't try to solve them because someone (Fabian probably?) tried to solve them but apparently stopped at some point (judging by the comment).